private/buf/cmd/buf/command/beta/price/price.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package price
+
+import (
+	"context"
+	"fmt"
+	"text/template"
+
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	"github.com/bufbuild/buf/private/buf/buffetch"
+	"github.com/bufbuild/buf/private/bufpkg/bufmodule/bufmodulestat"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/bufbuild/buf/private/pkg/command"
+	"github.com/bufbuild/buf/private/pkg/protostat"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	disableSymlinksFlagName     = "disable-symlinks"
+	teamsPerTypeDollarsPerMonth = float64(0.50)
+	proPerTypeDollarsPerMonth   = float64(1)
+	tmplCopy                    = `Current BSR pricing:
+
+  - Teams: $50/month per 100 types
```

Comment:
This information seems like it should be downloaded from the server. If it's built-in to the CLI, then someone on an old version of the CLI cannot get accurate pricing projections in the face of any tweaks or changes to pricing in the future.

Maybe a better way to do this is to send all of the stats to a simple BSR endpoint to compute the estimated price?

Reply:
Yea that's likely a better way, but that's more time than we want to allocate to this now, this gets this across the line and out there in under an hour's worth of work. We mention that the pricing information may not be the latest in the copy.